QUICK_START_GUIDE.md

```diff
@@ -2,135 +2,144 @@
+     - Rotated detection
 
-   Note that the variable `TASK_ALGO_DIR` is set in this example for simplicity and will not be used in scripts.
+   - `external/anomaly/init_venv.sh` can be used to create a virtual environment for the following task types.
+     - Anomaly_classification
+     - Anomaly_detection
+     - Anomaly_segmentation
 
-5. Create and activate a virtual environment for the chosen task, then install the `ote_cli`.
-   Note that the virtual environment directory may be created anywhere on your system.
-   The `./cur_task_venv` is just an example used here for convenience.
+1. Create and activate a virtual environment for the chosen task, then install the `ote_cli`. The following example shows that creating virtual environment to the `.venv_mpa` folder in your current directory for detection task.
 
    ```bash
-   bash $TASK_ALGO_DIR/init_venv.sh ./cur_task_venv python3.8
-   source ./cur_task_venv/bin/activate
-   pip3 install -e ote_cli/ -c $TASK_ALGO_DIR/constraints.txt
+   # create virtual env.
+   $ external/model-preparation-algorithm/init_venv.sh .venv_mpa
+   # activate virtual env.
+   $ source .venv_mpa/bin/activate
+   # install 'ote_cli' to the activated virtual env.
+   (mpa)...$ pip3 install -e ote_cli/ -c external/model-preparation-algorithm/constraints.txt
    ```
 
-   Note that `python3.8` is pointed as the second parameter of the script
-   `init_venv.sh` -- it is the version of python that should be used. You can
-   use any `python3.8+` version here if it is installed on your system.
+   > **_note_** that during installation of `ote_cli` the constraint file
+   > from the chosen backend folder is used to avoid breaking constraints.
 
-   Also note that during installation of `ote_cli` the constraint file
-   from the chosen task folder is used to avoid breaking constraints
-   for the OTE task.
-
-6. When `ote_cli` is installed in the virtual environment, you can use the
-   `ote` command line interface to perform various actions for templates related to the chosen task type, such as running, training, evaluating, exporting, etc.
+1. Once `ote_cli` is installed to the virtual environment, you can use the
+   `ote` command line interface to perform various commands for templates related to the chosen task type, described in [OTE CLI commands](#ote-cli-commands) on that virutal environment.
 
 ## OTE CLI commands
 
-### ote find
+### Find
 
-`ote find` lists model templates available for the given virtual environment.
+`find` lists model templates available for the given virtual environment.
 
 ```
-ote find --root $TASK_ALGO_DIR
-```
-
-Output for the mmdetection used in the above example looks as follows:
+usage: ote find [-h] [--root ROOT] [--task_type TASK_TYPE] [--experimental]
 
+optional arguments:
+  -h, --help              show this help message and exit
+  --root ROOT             A root dir where templates should be searched.
+  --task_type TASK_TYPE   filter with the task type (e.g., classification)
```

Comment:
How about showing the overall candidates of `--task_type`? Maybe users aren't aware of overall candidates.

Reply:
thanks for the opinion.

Reply:
updated. please check it again.